pep517/wrappers.py

```diff
@@ -143,25 +181,40 @@ def _call_hook(self, hook_name, kwargs):
         # letters, digits and _, . and : characters, and will be used as a
         # Python identifier, so non-ASCII content is wrong on Python 2 in
         # any case).
+        # For backend_path, we use sys.getfilesystemencoding.
         if sys.version_info[0] == 2:
             build_backend = self.build_backend.encode('ASCII')
         else:
             build_backend = self.build_backend
+        extra_environ = {'PEP517_BUILD_BACKEND': build_backend}
+
+        if self.backend_path:
+            backend_path = os.pathsep.join(self.backend_path)
+            if sys.version_info[0] == 2:
+                backend_path = backend_path.encode(sys.getfilesystemencoding())
```

Comment:
This may fail if a Unix filesystem erroneously reports that it's ASCII and someone tries to use a non-ASCII path. But I don't know what the correct thing to do on Python 2 in that case is (Python 3 has `os.fsencode()`). So we may as well cross our fingers and hope people use either Python 3 or plain ASCII paths.